src/OpenLoco/Objects/TrainStationObject.cpp

```diff
@@ -29,4 +29,29 @@ namespace OpenLoco
         Ui::Point rowPosition = { x, y };
         ObjectManager::drawGenericDescription(context, rowPosition, designed_year, obsolete_year);
     }
+
+    std::vector<TrainStationObject::CargoOffset> TrainStationObject::getCargoOffsets(const uint8_t rotation, const uint8_t nibble) const
```

Comment:
probably not worth for this PR but would be good to make `nibble` type safe, to avoid accessing/setting the upper 4 bits mistakenly